tests/test_browser_portal_lifecycle_architecture.py

```
@@ -0,0 +1,91 @@
+   not blind `webView.reload()`.
+"""
+
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+
+
+def repo_root() -> Path:
+    result = subprocess.run(
+        ["git", "rev-parse", "--show-toplevel"],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)